src/app/options.cpp

```diff
@@ -312,6 +434,27 @@ QBtCommandLineParameters parseCommandLine(const QStringList &args)
             else if (arg == CONFIGURATION_OPTION) {
                 result.configurationName = CONFIGURATION_OPTION.value(arg);
             }
+            else if (arg == SAVE_PATH_OPTION) {
+                result.savePath = SAVE_PATH_OPTION.value(arg);
+            }
+            else if (arg == PAUSED_OPTION) {
+                result.addPaused = PAUSED_OPTION.value(arg);
```

Comment:
You're forgetting about environment variables being used to specify options. It's definitely necessary to have `TriStateBoolOption` in order for that to work. Not to mention there's my earlier comment about how using `TriStateBoolOption` unambiguously informs people reading the code that it behaves differently than the other bool options, which I think vastly improves comprehensibility.